tools/test/travis-ci/scancode-evaluate.py

```diff
@@ -110,6 +103,29 @@ def license_check(directory_name, file):
             userlog.warning("File: " + offender['file']['path'][len(directory_name):] + " " + "reason: " + offender['reason'])
     return len(offenders)
 
+
+def spdx_check(offenders, license_offender):
+        """ Parse through list of licenses to determine whether licenses are permissive
+                @input list of offender, individual offender dict
+                @output none
+        """
+        found_spdx = False
+        # iterate through licenses, stop once permissive license has been found
+        for i in range(len(license_offender['file']['licenses'])):
```

Comment:
The `for` statement will automatically iterate over elements of a sequence. So, instead of explicitly calling `range(len(...` you can just have something like:

```
for license in license_offender['file']['licenses']:
    license['category'] == 'Permissive':
    ...
```